test/JDBC/expected/todatetimeoffset.out

```diff
@@ -0,0 +1,204 @@
+Select todatetimeoffset('0000-04-22 16:23:51.766890','+12:00')
```

Comment:
Please add test case for each exception condition. Test cases for boundary values (min/max/0), invalid values, data types and formats. Also, add test cases where values are coming from a table, input is expr, output is fed to expr etc.

Reply:
Added